src/electron/go_vpn_tunnel.ts

```diff
@@ -287,6 +290,7 @@ async function checkConnectivity(config: ShadowsocksSessionConfig) {
   args.push('-proxyPort', `${config.port}`);
   args.push('-proxyPassword', config.password || '');
   args.push('-proxyCipher', config.method || '');
+  args.push('-proxyPrefix', encodeURI(config.prefix || ''));
```

Comment:
It would be nice to have a single `-proxyConfig` flag with the json object, so we don't need to know about the fields here.